main.py

```diff
@@ -0,0 +1,51 @@
+from flask import Flask, render_template, request, redirect, url_for
+import pandas as pd
+import json
+import plotly
+import plotly.express as px
+from plotly.subplots import make_subplots
+import plotly.graph_objects as go
+ 
+app = Flask(__name__)
+
+df = pd.read_csv('women_EU2.csv', sep=',')
+
+@app.route('/', methods=['GET','POST'])
+def search():
+	df = pd.read_csv('women_EU2.csv', sep=',')
+	if request.method == 'POST':
+		country_name = request.form.get('country')
+		return redirect(f"/{country_name}")
+		#return redirect(url_for("country_plot", country=country_name))
+	return render_template('base.html', table=df)
+
+
+
```

Comment:
Try not to leave empty lines like this without a reason (in that case, usually one line is enough). You can install a code formatter in your editor to fix this for you.